sources/packages/api/src/services/education-program-offering/education-program-offering.service.models.ts

```diff
@@ -10,17 +8,11 @@ export class EducationProgramOfferingModel {
   studyEndDate: Date;
   offeringDelivered: string;
   offeringIntensity: OfferingIntensity;
-  get studyDates(): string {
-    if (this.studyStartDate === null) {
-      return "No program dates";
-    } else {
-      return `${dayjs(this.studyStartDate).format(
-        EXTENDED_DATE_FORMAT,
-      )} - ${dayjs(this.studyEndDate).format(EXTENDED_DATE_FORMAT)}`;
-    }
-  }
 }
-
+export class PaginatedOffering {
```

Comment:
Can we change it to be a generic interface and move to a generic place. I believe that this will be used for many other paginated results.

Reply:
updated as discussed